src/data-planes/views/DataPlaneListView.vue

```diff
@@ -0,0 +1,654 @@
+                      class="k-input"
+                      @change="(event) => updateVisibleTableHeaders(event, item.tableHeaderKey)"
+                    >
+
+                    {{ item.label }}
+                  </label>
+                </KDropdownItem>
+              </div>
+            </template>
+          </KDropdownMenu>
+
+          <KButton
+            class="add-dp-button"
+            appearance="primary"
+            :to="dataplaneWizardRoute"
+            data-testid="data-plane-create-data-plane-button"
+            @click="onCreateClick"
+          >
+            <span class="custom-control-icon">
+              +
+            </span>
+
+            Create data plane proxy
+          </KButton>
+
+          <KButton
+            v-if="$route.query.ns"
+            appearance="primary"
+            :to="$options.nsBackButtonRoute"
+            data-testid="data-plane-ns-back-button"
+          >
+            <span class="custom-control-icon">
+              ←
+            </span>
+
+            View All
+          </KButton>
+        </template>
+      </DataOverview>
+    </div>
+
+    <div class="data-planes-sidebar component-frame">
+      <DataPlaneEntitySummary
+        v-if="entity.basicData !== undefined && !entityIsLoading"
+        :entity="entity"
+        :insight-subscriptions="subscriptionsReversed"
+        :raw-entity="rawEntity"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+/** @typedef {import('../constants').ColumnDropdownItem} ColumnDropdownItem */
+
+import { mapGetters } from 'vuex'
+import { datadogLogs } from '@datadog/browser-logs'
+import { KButton, KDropdownItem, KDropdownMenu } from '@kong/kongponents'
+
+import { columnsDropdownItems, defaultVisibleTableHeaderKeys, getDataPlaneTableHeaders } from '../constants'
+import { Storage } from '@/utils/Storage'
+import { patchQueryParam } from '@/utils/patchQueryParam'
+import Kuma from '@/services/kuma'
+import { getSome, humanReadableDate, stripTimes } from '@/helpers'
+import { datadogLogEvents } from '@/datadogEvents'
+import {
+  checkKumaDpAndZoneVersionsMismatch,
+  compatibilityKind,
+  dpTags,
+  getDataplane,
+  getDataplaneInsight,
+  getDataplaneType,
+  getStatus,
+  getVersions,
+  parseMTLSData,
+  COMPATIBLE,
+  IMPORTANT_DATA_PLANE_TAG_LABELS,
+  INCOMPATIBLE_UNSUPPORTED_ENVOY,
+  INCOMPATIBLE_UNSUPPORTED_KUMA_DP,
+  INCOMPATIBLE_ZONE_CP_AND_KUMA_DP_VERSIONS,
+  INCOMPATIBLE_WRONG_FORMAT,
+} from '@/dataplane'
+import { PRODUCT_NAME } from '@/consts'
+import { getTableData } from '@/utils/tableDataUtils'
+import DataOverview from '@/components/Skeletons/DataOverview.vue'
+import DataPlaneEntitySummary from '@/data-planes/components/DataPlaneEntitySummary.vue'
+
+export default {
+  name: 'DataPlaneListView',
+
+  dataPlaneTypes: [
+    {
+      label: 'All',
+      type: 'all',
+    },
+    {
+      label: 'Standard',
+      type: 'Standard',
+    },
+    {
+      label: 'Gateway',
+      type: 'Gateway / BUILTIN',
+    },
```

Comment:
Are we missing provided gateways here?

Reply:
I’ve added an entry for `Gateway / PROVIDED`. I *assume* this is how the type would be constructed in the UI though I’m not sure because there is no mock data that features this type. I suppose the `gateway.type` field would be set to `'PROVIDED'` for those?

Reply:
A few questions:

- I’ve seen some test examples in Kuma that suggest that `gateway.type` isn’t typically all uppercase. However, this is the case in our mock data. If it’s lowercase, I’m going to change the mock data.
- “Gateway / BUILTIN” looks strange to me. I suggest a slightly different format for the UI: “Gateway (builtin)” (and “Gateway (provided)”). What do you think?

Reply:
These are the 2 different ways to run gateways: https://kuma.io/docs/1.8.x/explore/gateway/

Going to try to find enough time to start the 2 gateways and get you the API output. FTM I think it's fine to merge as is and open a follow up issue if it's addressed before 2.0